test/TimeZones.jl

```diff
@@ -37,13 +37,13 @@ if lowercase(get(ENV, "CI", "false")) == "true"
 
 
     # Compile tz source files with an extended max_year. An example from the FAQ.
-    warsaw = TimeZone("Europe/Warsaw")
+    warsaw = tz"Europe/Warsaw"
     @test last(warsaw.transitions).utc_datetime == DateTime(2037, 10, 25, 1)
     @test get(warsaw.cutoff) == DateTime(2038, 3, 28, 1)
     @test_throws TimeZones.UnhandledTimeError ZonedDateTime(DateTime(2039), warsaw)
 
     TimeZones.TZData.compile(max_year=2200)
-    new_warsaw = TimeZone("Europe/Warsaw")
+    new_warsaw = tz"Europe/Warsaw"
```

Comment:
Lets leave this line and the [above line](https://github.com/JuliaTime/TimeZones.jl/pull/64/files#diff-72390e967fd3f6573a6058eba616722aR40) the way the were.